test/Microsoft.AspNetCore.SignalR.Client.Tests/TestConnection.cs

```diff
@@ -146,6 +146,28 @@ public Task ReceiveBytesAsync(byte[] bytes)
             }
         }
 
+        public async Task<IList<string>> ReadAllSentMessagesAsync()
+        {
+            if (!_disposed.Task.IsCompleted)
```

Comment:
You can actually just write this as `Disposed.IsCompleted` because we already define the `Disposed` property above as `_disposed.Task` :)